brambling/static/brambling/sass/modules/_event-listing.sass

```diff
@@ -9,32 +9,16 @@
 	box-sizing: border-box
 	border-radius: $border-radius-base
 
-.datecard
-	position: absolute
-	top: 6px
-	left: 6px
-	text-align: center
-	width:  50px
-	height: 50px
-	border-radius: $border-radius-base
-	float: left
-	color: white
-	box-shadow: inset 0 0 0 1px rgba(0,0,0,.2)
-	background-color: rgba($yellow-dark,0.9)
-
-.datecard-month
-	margin: 5px 0 0
-	text-transform: uppercase
-	font-size: floor(($font-size-base * 1.25))
-	font-weight: $headings-font-weight-normal
-
-.datecard-day
-	margin: 0
-	line-height: 1em
-	font-size: floor(($font-size-base * 1.5))
-
 .event-listing
 	& > .relative-right
 		right: 6px
 	& > .relative-top
 		top: 6px
+
+	& > h3
```

Comment:
I'm not gonna change this because it matches the rest of the file and I don't think there's a compelling reason _not_ to do this...